packages/lesswrong/components/comments/CommentOnSelection.tsx

```diff
@@ -149,8 +150,15 @@ const SelectedTextToolbar = ({onClickComment, x, y, classes}: {
   x: number, y: number,
   classes: ClassesType,
 }) => {
+  const { captureEvent } = useTracking()
+
   return <div className={classes.toolbar} style={{left: x, top: y}}>
-    <CommentIcon onClick={ev => onClickComment(ev)}/>
+    <AnalyticsContext pageElementContext="selectedTextToolbar">
+      <CommentIcon onClick={ev => {
+        captureEvent("commentOnSelectionClicked");
```

Comment:
If we can pass the selected text into the second argument, seems maybe valuable.